### PR TITLE
Fixes permlink copying #1927

### PR DESCRIPTION
**Pull request by moogacs**
## What does this PR do?
fixing copying to permalink
## How do I test this PR?
click the button

### CODE REVIEW

**Comment by kgardnr** [comment]
Thanks for this @moogacs!

**Comment by moogacs** [comment]
@kgardnr it's still open ? or you have merged and closed it ?

**Comment by wyattjoh** [comment]
We have yet to review it @moogacs, I'll take a review shortly!

File: plugins/talk-plugin-permalink/client/components/PermalinkButton.js [hunk 1]
```diff
@@ -36,7 +36,9 @@ export default class PermalinkButton extends React.Component {
   };
 
   copyPermalink = () => {
+    this.permalinkInput.focus();
     this.permalinkInput.select();
+    this.permalinkInput.createTextRange();
```
Comment (wyattjoh) [inline plugins/talk-plugin-permalink/client/components/PermalinkButton.js:41]: It seems that this is not a valid method for input elements: https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement

**Review by wyattjoh — CHANGES_REQUESTED** [review]
From tests conducted locally, the existing implementation works in Firefox, Chrome, and Safari.

What browsers does this version work on that the current implementation doesn't?

File: plugins/talk-plugin-permalink/client/components/PermalinkButton.js (hunk 1 above)
Comment (moogacs) [reply plugins/talk-plugin-permalink/client/components/PermalinkButton.js:41]: So you mean before my PR it was fine ? 
it suppose an open task !

**Comment by kgardnr** [comment]
Ok so strangely I'm not able to replicate this issue somehow - I can't determine if it was fixed or not. Apologies for that @moogacs - I'm just going to close this - but let me know if you see it happen again!

**Pull request closed without merging**